tests/BCM56850/test_macsec_p2p_establishment.rec

```diff
@@ -0,0 +1,2300 @@
+2021-01-21.03:14:40.620302|c|SAI_OBJECT_TYPE_ACL_TABLE:oid:0x700000000058b|SAI_ACL_TABLE_ATTR_ACL_STAGE=SAI_ACL_STAGE_INGRESS|SAI_ACL_TABLE_ATTR_ACL_BIND_POINT_TYPE_LIST=1:SAI_ACL_BIND_POINT_TYPE_SWITCH|SAI_ACL_TABLE_ATTR_FIELD_ETHER_TYPE=true|SAI_ACL_TABLE_ATTR_FIELD_SRC_IP=true|SAI_ACL_TABLE_ATTR_FIELD_DST_IP=true|SAI_ACL_TABLE_ATTR_FIELD_L4_SRC_PORT=true|SAI_ACL_TABLE_ATTR_FIELD_L4_DST_PORT=true|SAI_ACL_TABLE_ATTR_FIELD_IP_PROTOCOL=true|SAI_ACL_TABLE_ATTR_FIELD_TUNNEL_VNI=true|SAI_ACL_TABLE_ATTR_FIELD_INNER_ETHER_TYPE=true|SAI_ACL_TABLE_ATTR_FIELD_INNER_SRC_IP=true|SAI_ACL_TABLE_ATTR_FIELD_INNER_DST_IP=true|SAI_ACL_TABLE_ATTR_ACL_ACTION_TYPE_LIST=4:SAI_ACL_ACTION_TYPE_ACL_DTEL_FLOW_OP,SAI_ACL_ACTION_TYPE_DTEL_INT_SESSION,SAI_ACL_ACTION_TYPE_DTEL_REPORT_ALL_PACKETS,SAI_ACL_ACTION_TYPE_DTEL_FLOW_SAMPLE_PERCENT
+2021-01-21.03:14:40.621822|c|SAI_OBJECT_TYPE_ACL_TABLE:oid:0x700000000058c|SAI_ACL_TABLE_ATTR_ACL_STAGE=SAI_ACL_STAGE_INGRESS|SAI_ACL_TABLE_ATTR_ACL_BIND_POINT_TYPE_LIST=1:SAI_ACL_BIND_POINT_TYPE_SWITCH|SAI_ACL_TABLE_ATTR_FIELD_ETHER_TYPE=true|SAI_ACL_TABLE_ATTR_FIELD_SRC_IP=true|SAI_ACL_TABLE_ATTR_FIELD_DST_IP=true|SAI_ACL_TABLE_ATTR_FIELD_L4_SRC_PORT=true|SAI_ACL_TABLE_ATTR_FIELD_L4_DST_PORT=true|SAI_ACL_TABLE_ATTR_FIELD_IP_PROTOCOL=true|SAI_ACL_TABLE_ATTR_ACL_ACTION_TYPE_LIST=2:SAI_ACL_ACTION_TYPE_DTEL_DROP_REPORT_ENABLE,SAI_ACL_ACTION_TYPE_DTEL_TAIL_DROP_REPORT_ENABLE
+2021-01-21.03:14:40.625759|a|APPLY_VIEW
```

Comment:
put apply view at the end of the log file to make sure everything will be processed